axlearn/common/flash_attention/gpu_attention.py

```diff
@@ -666,3 +673,96 @@ def _mha_backward(
 
 
 flash_attention.defvjp(_mha_forward, _mha_backward)
+
+
+def check_local_compute_capability(cc):
```

Comment:
Does this need to be a public fn? Also annotate types/returns/docstring etc?

Reply:
Done